blueprint/NICBlueprint.go

```diff
@@ -10,7 +12,22 @@ func CreateNICBlueprint() *NICBlueprint {
 	return &NICBlueprint{Blueprint: *CreateBlueprint("NIC")}
 }
 
-// SetNetwork sets NETWORK of a given VM.
-func (nb *NICBlueprint) SetNetwork(value string) {
-	nb.SetElement("NETWORK", value)
+// SetNetworkName sets NETWORK_NAME of a given VM.
+func (nb *NICBlueprint) SetNetworkName(value string) {
```

Comment:
I think that `NETWORK_NAME` doesn't exist. I think it's `NETWORK` and it sets network name.